src/Query/AST/Functions/Mariadb/JsonExists.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Syslogic\DoctrineJsonFunctions\Query\AST\Functions\Mariadb;
+
+use Doctrine\DBAL\DBALException;
+use Doctrine\DBAL\Platforms\MySqlPlatform;
+use Doctrine\ORM\Query\AST\Functions\FunctionNode;
+use Doctrine\ORM\Query\Lexer;
+use Doctrine\ORM\Query\Parser;
+use Doctrine\ORM\Query\SqlWalker;
+use Syslogic\DoctrineJsonFunctions\Query\AST\Functions\Mysql\JsonKeys;
+
+/**
+ * "JSON_EXISTS" "(" StringPrimary "," StringPrimary  ")"
+ */
+class JsonExists extends JsonKeys
```

Comment:
Thank you for your contribution.
I do think though it's better not to extend a MySQL implementation.

I find that MySQL and MariaDB are really going their separate ways and I do not like the fact that a MySQL definition is leading for a MariaDB query implementation. This could lead to unexpected behaviour when `JsonKeys` needs to be chanced.

Maybe abstacting away query generation is not a bad idea, but I'd like a completely different abstraction layer for this then. For now, copying and changing this class into the JsonKeys methods is a good enough solution to me.

Reply:
Hi @Hikariii , I totally agree with you.

So for now, I copied the JsonKeys class and only replaced the parts that are needed for JsonExists

Let me know if more changes are required